homeTask_10/task1.js

```diff
@@ -0,0 +1,5 @@
+function busSchedule(message, ms) {
```

Comment:
function name should represent an action - "What to do?"

Reply:
done